ui/app/models/ldap/library.js

```diff
@@ -60,6 +60,10 @@ export default class LdapLibraryModel extends Model {
   })
   disable_check_in_enforcement;
```

Comment:
Looking at the [docs](https://developer.hashicorp.com/vault/api-docs/secret/ldap#disable_check_in_enforcement) I believe the default here should be `Enabled` since `disable_check_in_enforcement = false` means not disabled

Reply:
Ahhh that's a good catch! I was confused by the language it seems because I have `false` mapped to `Disabled` in the serializer 😞

Reply:
Yeah! I've come across something similar with false defaults in pki that were mapped to `disable` params. It's also why I've avoided changing them in the serializer and just handle the viewing logic client side because it makes it easier to debug later